server/import.go

```diff
@@ -131,25 +132,45 @@ func createECCSeed(ek tpm2.Public) (seed, encryptedSeed []byte, err error) {
 	if err != nil {
 		return nil, nil, err
 	}
-	priv, x, y, err := elliptic.GenerateKey(curve, rand.Reader)
+
+	ecdsaPriv, err := ecdsa.GenerateKey(curve, rand.Reader)
```

Comment:
We're doing ecdh here so you should use the utilities in crypto/ecdh instead

Reply:
We need EC points {x, y} to compute ECC seed in the following operations. Using `crypto/ecdh` utilities is an ideal way, however they don't provide a method to expose the EC points {x, y} in public key. 

crypto/ecdsa provides a method to do that, also there are methods which convert each key type to their counterparts in the the ecdh package. See line 141 that converts an ecdsa private key to its corresponding ecdh private key.

Reply:
Makes sense. LGTM